### ISSUE TITLE
Device Additions Requests

### ISSUE THREAD

**Issue by SkewedZeppelin** [Divested-Mobile/DivestOS-Build#68]
Please only comment if your device has or had official LineageOS support.

**Comment by taivlam** [comment]
Could you add the Samsung Galaxy Tab S6 Lite (`gta4xlwifi`)?
The device is currently [supported](https://wiki.lineageos.org/devices/gta4xlwifi/) by LineageOS.

**Comment by SkewedZeppelin** [comment]
There is no order for these.

**Comment by dennyamarojr** [comment]
Could you add Motorola x4 (payton) to the devices supported? LineageOS has the support for it

**Comment by b3pio** [comment]
Would you mind adding support for Galaxy A5 (2017) '[a5y17lte](https://wiki.lineageos.org/devices/a5y17lte/)' Tad?

**Comment by Landiks** [comment]
Are there any dates for the devices on the list? I can test it for Miatol.

**Comment by SkewedZeppelin** [comment]
None of these devices on the list have even been added, let alone compiled.

It takes 2-4 hours for me to add a device.
And adding devices then adds an hour of compile every build cycle.
DivestOS build cycle is curently a little over 2 whole days on a 5900X with 64GB
of RAM and 1.2TB backing SSD and another 1TB cache SSD and a 1TB artifacts
drive.

Adding every device possible is unsustainable.
My focus right now is fixing up and improving the devices I currently support
and getting Android 12 up to parity.

More devices will eventually be added, but not right now.

**Comment by b3pio** [comment]
Tad what's the best way to support/assist you with DivestOS devices generally speaking, financially (donations etc) or compiling or.. ?

**Comment by SkewedZeppelin** [comment]
Pull requests actually adding the devices would be most helpful.
But it involves a litany of steps, many of which are currently undocumented.

**Comment by b3pio** [comment]
Any chance to add the Xiaomi Mi A2 ([jasmine_sprout](https://wiki.lineageos.org/devices/jasmine_sprout/))? I have one and can probably help in some way. Thanks!

**Comment by jontaix** [comment]
Lenovo Z5 Pro GT ([heart](https://wiki.lineageos.org/devices/heart/)) has official LineageOS support.

**Comment by onkq** [comment]
Samsung Galaxy Tab S5e (LTE) [[gts4lv](https://download.lineageos.org/gts4lv)] is a very popular tablet. Android 12 support from LineageOS already.

**Comment by SkewedZeppelin** [comment]
Yes, gts4lv is a bit higher priority since there are so few tablets.

**Comment by mikitakahazekura** [comment]
Could you please add the Motorola Edge (racer)? The phone The phone is currently [supported](https://wiki.lineageos.org/devices/racer/) by LineageOS.
Thanks for your work.

**Comment by TeaqariaWTF** [comment]
Poco X3 (Surya) [lineage 19.1 support](https://download.lineageos.org/surya)

**Comment by AkechiShiro** [comment]
Please add device Poco F3 : https://wiki.lineageos.org/devices/alioth/ LineageOS 18.1/19.1 support.

**Comment by SkewedZeppelin** [comment]
@AkechiShiro
alioth is already supported on 18.1 and 19.1 will be available soon

**Comment by Blackmine57666** [comment]
Could you add a J3 2016 ? [Not officially supported](https://forum.xda-developers.com/t/rom-lineageos-14-1-for-samsung-galaxy-j3-2016-sm-j320fn-f-g-m-unofficial.3667015/) (lin14)
Also this is not a phone but could you add a SM-T580 (gtaxlwifi). [Not officially supported ](https://forum.xda-developers.com/t/lineageos-18-1-for-sm-t580-gtaxlwifi-and-sm-p580-gtanotexlwifi.4274293/) (lin19)
PS : I'd appreciate a documentation (or a link to a tutorial) to port new devices.

**Comment by AkechiShiro** [comment]
@SkewedZeppelin can I help work on the support for 19.1 ? I want to help, I tried to help LineageOS folks but met much toxicity...

**Comment by SkewedZeppelin** [comment]
@AkechiShiro 

DivestOS 19.1 has been stable for a month now.
I've already dropped nearly all 17.1 and 18.1 targets that are working on 19.1.

lmi 19.1 was tested not booting however, given that alioth is nearly identical,
it is also likely to not work. Nothing I can do as I don't have such device to
test with.

**Comment by AkechiShiro** [comment]
Booting on the alioth device was not tested, if LOS 19.1 works alright for alioth, then what are the risks that booting under DivestOS 19.1 doesn't work ? I could do some tests, but I just hope it won't hard brick my phone

**Comment by thzgr** [comment]
Since the best way to assist would be opening PRs adding the devices, is there any chance of us getting a step by step documentation to build DivestOS for a new device?
I've found [this](https://divestos.org/index.php?page=build) build guide. Is it enough or are there missing steps?

I'd like to have DivestOS working for montana and use it as my daily driver, I have the device and a desktop with 16GB of RAM, a R5 1600 which is 6/12 and 1TB HDD + 500GB SSD available. I know that maybe it would take a long time (5-6 hours?) but I'm able to at least test if it works properly.

**Comment by SkewedZeppelin** [comment]
@thzgr

If you set up everything according to that page, source correct vendor blobs,
and simply let breakfast fetch the device trees for you it will compile.

You can then take the .repo/local_manifests/roomservice.xml contents and merge
them appropriately with the corresponding manifest and again update the local
manfiest.

But you will miss out on the kernel patching/hardening, verified boot/signing
changes, and things like WebView updates.

You can grep for grep comments in the repo and you'll see a few you have to run
and update the following lines appropriately.

You can see an example here recently:
https://github.com/Divested-Mobile/DivestOS-Build/commit/0eaca57fa61d18547772b3a7f5dc69cdaf53383c

That computer should easily compile in two hours.

**Comment by pixincreate** [comment]
Support for [DRG](https://wiki.lineageos.org/devices/DRG) (Nokia 6.1 Plus)? It's officially supported by LineageOS BTW which is currently on LOS 19.1

**Comment by opusforlife2** [comment]
Considering it has had official LOS for almost a year, I'm surprised no one has requested Pixel 5a (barbet) yet!

**Comment by SkewedZeppelin** [comment]
@opusforlife2 
if you have a Pixel I strongly recommend you use GrapheneOS instead.

**Comment by ByJumperX4** [comment]
Could support for raphael (Redmi K20 Pro / Mi 9T Pro) be added ? It's not listed on the supported devices on the lineage website but [this repo from LineageOS with 18.1 and 19.1](https://github.com/LineageOS/android_device_xiaomi_raphael) exists and was updated last month.

**Comment by SkewedZeppelin** [comment]
@ByJumperX4 
raphael was added here 0b4ad0e7cc28c1d791953a7621061dcb203c428e and never compiled, it was later removed here 697bed18fbea3fe34164e2d5ef82829a50888e52

**Comment by ByJumperX4** [comment]
> @ByJumperX4 raphael was added here [0b4ad0e](https://github.com/Divested-Mobile/DivestOS-Build/commit/0b4ad0e7cc28c1d791953a7621061dcb203c428e) and never compiled, it was later removed here [697bed1](https://github.com/Divested-Mobile/DivestOS-Build/commit/697bed18fbea3fe34164e2d5ef82829a50888e52)

oh okay 😔

**Comment by SkewedZeppelin** [comment]
@AkechiShiro
alioth and lmi share a device tree and kernel.
I guarantee if lmi isn't booting, that alioth won't either.
The risk is simply having to flash another OS again.
Brick chance is near impossible.

**Comment by SkewedZeppelin** [comment]
Pull requests actually adding the devices would be most helpful.
But it involves a litany of steps, many of which are currently undocumented.

**Comment by AkechiShiro** [comment]
But I don't understand, it seems that 19.1 has been reported very likely working for a month now for Alioth, has it broke just really recently on lmi for booting ?

**Comment by SkewedZeppelin** [comment]
@AkechiShiro 
this is something weird going on: https://github.com/Divested-Mobile/DivestOS-Build/issues/155

please ignore those

**Comment by cryptoo01** [comment]
Please add support.
                Thank you!

**Comment by TheHawaiianKoala** [comment]
Kindly add support for Samsung Galaxy Note10+ (d2s). It already has LOS 19.1 and the maintainer is working on LOS 20 unofficially. [Link](https://wiki.lineageos.org/devices/d2s/)

**Comment by OkyDooky** [comment]
I would like to request the Galaxy Note 3 T-Mobile variant (hltetmo). I have been needing to do a manual OS upgrade for a long time from an older Lineage+MicroG version, so it would be cool to use a more maintained and secure version of the OS.
Thanks for making and maintaining this. It's cool that people are finally starting to really recognize its existence and value.

**Comment by OkyDooky** [comment]
@opusforlife2 
>It doesn't have LOS's network traffic/speed indicator. It's "not a privacy or security issue", so not cared about, apparently.

Being able to see basic network usage in real-time is exactly a privacy & security issue, since most malware/spyware gets away with what they do because you don't know anything's being done in the first place. This feature is definutely more than just a convenience.
@SkewedZeppelin I might be getting a Pixel 4 XL, due to it fitting my needs (horsepower, affordable, non-5G), which means GrapheneOS will not be a good option, since they ended official support for the device last month. CalyxOS will be doing the same in about two (2) years. This means DivestOS would be the best long-term option for even Pixel users, as it'll be very secure and continue to offer "official" updates.

**Comment by SkewedZeppelin** [comment]
@OkyDooky 
I strongly recommend against buying an insecure end of life Pixel 4 and instead getting a Pixel 6a, which has another 4.5+ years of support left.

**Comment by OkyDooky** [comment]
@SkewedZeppelin I barely even want to buy a Pixel, since I have more concern over Google than "rouge" "bad actors" trying to steal my identity (not a practical issue for me, currently - and there's not much I can currently reasonably do about CIA/NSA/Mossad without giving up A LOT of convenience). But, it is one of two* that can meet my needs/criteria.
I appreciate the concern, but I am currently not planning on compromising on certain factors like getting a 5G device (which getting a Pixel 5 or above would do).
>it'll be very secure

I'm talking about your OS versus other Lineage builds, not the device.
Also, I appreciate being honest and using "secure" in assumptive "perfect" sense. We're all getting tired of false advertising and seeing this is refreshing.

*the other device is a Galaxy S9+, but it has to be the Exynos version, because the US Snapdragon one cannot be unlocked. This means that I'll also have to switch networks just to use it, since my current provider decided not to support any devices that don't support the VoLTE HD Calling feature. So, I'm currently a bit torn between convenience + a bit more power **vs** not-Google + microSD card support.

**Comment by sbe-arg** [comment]
> @OkyDooky
> 
> > getting a Pixel 4 XL
> > it'll be very secure
> 
> I strongly recommend against buying an insecure end of life Pixel 4 and instead getting a Pixel 6a, which has another 4.5+ years of support left. https://divestos.org/index.php?page=patch_levels#osSecurity

Pixel 6a with graphene is the best bet for money saving and security focus as today. Then will be 7a and 8a so on XD

I use as main a 6a with graphene is been amazing 0 issues so far
I use divest as explorer on a nexus 4, nexus 7 and praying for a fix of the nexus 7 2013 deb (lte) builds one day to revive that tablet for the kids.

**Comment by DIXLOR15** [comment]
@SkewedZeppelin  https://download.lineageos.org/ginkgo Ginkgo get Offical support

**Comment by mohammadRezaeian** [comment]
Could you please add device Motorola edge 20 pro (pstar)
Thanks

**Comment by Disposable-account** [comment]
Can you add the Motorola moto g 5G / one 5G ace at https://wiki.lineageos.org/devices/kiev/

**Comment by silverwind63** [comment]
Could you add support for haydn,it has been supported by Lineage OS 19.1

**Comment by Filbuntu** [comment]
DivestOS would be great. Thanks a lot for all the work on it. 
Will [Galaxy S10+ (beyond2lte) ](https://wiki.lineageos.org/devices/beyond2lte/)be supported soon? Thank you for the kind consideration!

**Comment by owlysheaven** [comment]
First of all thank you for your good work!
Can you please support Xiaomi Mi8 codename: Dipper. It has LOS 20 [https://download.lineageos.org/devices/dipper/builds](url)

And Xiaomi Mi Mix 2 codename: chiron. It also has LOS 20 [https://download.lineageos.org/devices/chiron/builds](url)

Thanks in advance!

**Comment by SirRGB** [comment]
Moto G5 aka cedric has official Lineage 18.1 support, would be great to have another reliable rom, but more on the private side :+1:

**Comment by SkewedZeppelin** [comment]
there have been 9 device additions:
20.0 additions: akatsuki, dipper, equuleus, polaris, ursa, lemonades (failed)
19.1 additions: kirin, mermaid, apollon

**Comment by SkewedZeppelin** [comment]
9 more additions:
18.1 additions: serranodsdd, jactivelte, jfvelte, hlte, jasmine_sprout, platina, twolip, wayne, whyred

**Comment by owlyhub** [comment]
First of all, I want to thank you for adding support for Dipper. 
However, I am experiencing an issue with the camera where it crashes and reboots the phone every time I try to use it."
I hope it can be fixed. On lineageos there is no problem with the cam.

**Comment by SkewedZeppelin** [comment]
@owlyhub that is expected as the same issue as https://github.com/Divested-Mobile/DivestOS-Build/issues/37

**Comment by girlwithnoname** [comment]
Could you please add Moto G7 Power? It has official LOS 20 support: https://wiki.lineageos.org/devices/ocean/

**Comment by OkyDooky** [comment]
I was about to ask for that, as well. Thank you for requesting it.

**Comment by SkewedZeppelin** [comment]
ocean is already in the list, please check the list before asking

**Comment by OkyDooky** [comment]
Which list? I checked https://divestos.org/pages/devices and did a Find In Page search for both "ocean" and "power" in separate searches, both coming up with exactly zero matches. There is #65 that was closed to be tracked here, but that's all I can see.

**Comment by SkewedZeppelin** [comment]
@OkyDooky 
The list linked at the very top of this issue
> Currently pending additions: https://github.com/Divested-Mobile/DivestOS-Build/blob/master/TODO-device_additions

**Comment by MajesticTwelve12** [comment]
Huawei Honor 5X (Kiwi) KIW-L24
It runs LineageOS 17.1 (Android 10), but it is not being developed no more and frozen.

https://wiki.lineageos.org/devices/kiwi/
https://github.com/LineageOS/android_device_huawei_kiwi
Thanks.

**Comment by ThrowawayAccount-1** [comment]
I would also say something similar like enduring but for this phone https://wiki.lineageos.org/devices/kiev/variant2/ and/or for other phones outdated. Also due to the reddit drama do you think of making a lemmy instance due to its FOSS nature and due to its growth currently? Im also curious on how often new phones are added like if there is a period of time like 1-3 months until new phones are added or if there able to be added when people make builds for them.

**Comment by overtube** [comment]
Hi, guys! pls add that device, his great
![specmotog31](https://github.com/Divested-Mobile/DivestOS-Build/assets/15115920/dfb70b0d-950f-4b9d-b373-924529044f0d)

**Comment by sbe-arg** [comment]
@SkewedZeppelin can you check on fixing the nexus 7 devx 2013? https://download.lineageos.org/devices/debx/builds they seem supported again

**Comment by SkewedZeppelin** [comment]
@sbe-arg 
debx is high up on my todo, should hopefully be in next update

**Comment by SkewedZeppelin** [comment]
@sbe-arg 
debx is up

**Comment by sbe-arg** [comment]
> @sbe-arg 
> debx is up

Amazing Ill test it today/tomorrow

**Comment by sbe-arg** [comment]
debx works like a charm thanks a million

**Comment by zynpachi** [comment]
Mi8917 -> https://wiki.lineageos.org/devices/Mi8917/

Same as Mi8937, which is supported already. All trees are basically the same, and I already tried Mi8937 DOS and works fine.

**Comment by ippocratis** [comment]
Would love to see divestOS for curtana (redmi note 9s)

The device belongs to a family of xiaomi devices sharing almost identical hardware (typical xiaomi)

POCO M2 Pro / Redmi Note 9S / Redmi Note 9 Pro / Redmi Note 9 Pro Max / Redmi Note 10 Lite

Under the "unified" name miatol

Latest lineageos:
https://download.lineageos.org/devices/miatoll/builds

Thanks
Keepup

**Comment by ippocratis** [comment]
Poco x5 pro "redwood" 
This is tricky
Not eol as it is a 2023 release
no official los (yet)
But it is the only 2023 xiaomi device with a released kernel source together with the poco f5 

Device tree  repo:https://github.com/xiaomi-redwood-devs
Kernel source:
https://github.com/MiCode/Xiaomi_Kernel_OpenSource/tree/redwood-s-oss

**Comment by SkewedZeppelin** [comment]
@IzumiSenaSora

yep, those are on my todo list, made a test build a few weeks back

**Comment by SkewedZeppelin** [comment]
@onkq 
Why do I support Pixels when GrapheneOS exists? Because the projects have completely different goals and DivestOS offers _many_ things that GrapheneOS does not.

And yes, tablets are higher priority, but also so is supporting the devices I already support.

**Comment by praticks** [comment]
Hi @SkewedZeppelin , Could you please add device Redmi Note 10 Pro / Max (Sweet) 
It's has official support of LineageOS 20.  https://wiki.lineageos.org/devices/sweet/
Thanks you.

**Comment by knightcole** [comment]
@SkewedZeppelin speaking of Pixels, please add the Google Pixel 7a (Lynx).
LineageOS support exists for it: https://wiki.lineageos.org/devices/lynx/

Thanks!

**Comment by noirge** [comment]
Any advice what I could do with my Samsung Galaxy S6? Only found an unofficial LineageOS on XDA. Maybe the best choice?

**Comment by SkewedZeppelin** [comment]
@noirge 
unofficial systems can be harmfully outdated or filled with junk just the same stock can
I strongly advise against them

sell the device if possible and find a more suitable replacmenet

**Comment by mohammadRezaeian** [comment]
Hi
Maybe my question sounds silly
I want to know if it's possible to fork from other ROMS and use it instead of LineageOS ?

( If they have followed the things you have said )

**Comment by SkewedZeppelin** [comment]
@mohammadRezaeian 
yes, but reviewing and actually testing to see if it meets the bar and works well can easily take five hours of work, so I don't do it very often

**Comment by mohammadRezaeian** [comment]
Isn't it even possible for you to fork it and create a build , and we test it ?

**Comment by ReduxFlakes** [comment]
Hi, Xiaomi 11 Lite 5G NE (lisa) is already planned for 20.0 among other 2 similar devices. Is there any ETA available for stable release or public testing builds? Thank you!

**Comment by michalszmidt** [comment]
Hello,

I've just installed DivestOS on my relatives enchilada and I'm just so impressed! Thank you for maintaining this project, it's absolutely awesome! I really liked the security and privacy features. :heart: It has everything I need.

Could you consider supporting Sony Xperia XZ1 Compact?
4/32 with sdcard slot, msm8998, 2018

I'm aware that you prefer newer devices but also I don't really like the xperia xz2 compact design and the fact it lacks microjack. I have huge problem with big devices (e.g. I consider enchilada as a big one) I need to have small one.

I know it has [only unofficial LOS20 port](https://xdaforums.com/t/rom-lineageos-20-0-unofficial-1-2-2023-05-10.4500913/), but:

> if you go through and ensure there is no funky root

From XDA thread:
They are not interested in it
_"Note: There is not going to be a LOS su addon anymore. If you need root, flash a root manager of your choice. No guaranteed support from LOS for this"._

> silly overclocking

Not mentioned, shouldn't be there.

> SELinux is enforcing

From XDA thread:
They care, 
_"Don't flash ROMs or Magisk modules which set SELinux to permissive or you welcome malware on your device!"_

>  and the maintainer is responsive and passionate about their devices

at least [derfelot](https://github.com/derfelot) and [flamefire](https://github.com/Flamefire) are

Sources:
- [android_device_sony_lilac](https://github.com/whatawurst/android_device_sony_lilac)
- [android_kernel_sony_msm8998](https://github.com/whatawurst/android_kernel_sony_msm8998)
- [android_device_sony_yoshino-common](https://github.com/whatawurst/android_device_sony_yoshino-common)
- [android_vendor_sony_yoshino-common](https://github.com/whatawurst/android_vendor_sony_yoshino-common)
- [android_vendor_sony_lilac](https://github.com/whatawurst/android_vendor_sony_lilac)

**Comment by SkewedZeppelin** [comment]
@michalszmidt 
maybe eventually
and yes, flamefire sets a great example for unofficial ports, they also help with the monthly ASB backports for 17.1

**Comment by ourslfs** [comment]
Hi! Can we expect support for redmi note 10 pro? https://wiki.lineageos.org/devices/sweet/

**Comment by d4f5409d** [comment]
Do you have an ETA for the Fairphone 5?

**Comment by SkewedZeppelin** [comment]
@d4f5409d when LineageOS or CalyxOS supports it, I'll add it.

**Comment by mohammadRezaeian** [comment]
> @mangoheini I already build 20.0 for vayu but had to pull it because Wi-Fi is broken on it and no one will help debug it properly.

how we can help you for debug vayu 20 ?

**Comment by KG93** [comment]
> @mangoheini I already build 20.0 for vayu but had to pull it because Wi-Fi is broken on it and no one will help debug it properly.

@SkewedZeppelin Due to AVB i'm currently stuck on DivestOS 20 (8.11) anyways. If i can help with some specific information, let me know. I could also test a build generated from the new LineageOs 20 release...

Trying proper debugging the wifi would also be an option. Would something along the lines of https://source.android.com/docs/core/connect/wifi-debug be the right course?

**Comment by 0xC4aE1e5** [comment]
Bringing it up again to add support for lynx (P7a). I am planning on getting this phone, so it would be nice.

**Comment by JensTheCoder** [comment]
> * Does GrapheneOS still lead in terms of security and testing? Yes.
> * Do I recommend it? Yes.

@SkewedZeppelin 
Regarding Pixel 6a, a device actively supported by DivestOS and GrapheneOS. Why do you recommend another OS in comparison to your own OS? I have been using GrapheneOS from 2020 to 2023. More than three years. I am pretty aware of things the team has done to it. Nonetheless, I also am aware of the work you have done with DivestOS. If you don't trust your own system, what is missing?

**Comment by SkewedZeppelin** [comment]
@JensTheCoder 
6a is my daily driver on DivestOS.

But GrapheneOS has a team of people and millions of dollars in funding.
They can update and test more extensively than I can.

I already tried for grants from FUTO and NLNet this year and was declined by both.
FUTO was "no."
NLNet needed there to be a maintainer that was in EU to qualify, despite the website saying otherwise.

**Comment by less-ly** [comment]
Hello!

Can you, please, consider adding [Xiaomi Mi A1](https://wiki.lineageos.org/devices/tissot/)?
It has an ongoing LineageOS 20 support, features 4.9 Linux kernel, and has a straightforward fastboot unlocking mechanism. It can be worth a shot.

BR,
less-ly.

**Comment by dktzde** [comment]
Device request: Samsung A3 2017 (A320F)

The Samsung Galaxy A3 2017 is a small, lightweight smartphone that is still popular with many users. I really like it.

Unfortunately, there is no official build. 
You can find builds of LineageOS here: 
https://github.com/a3y17lte

**Comment by Pit1s7** [comment]
Can now finally add Oneplus 11 salami - build is now officially supported by lineageos

**Comment by RizzoV** [comment]
Any plans on supporting the Pixel 8 (Pro) with the upcoming 21.0 builds?

**Comment by SkewedZeppelin** [comment]
@RizzoV 
of course, once I migrate to 21 I'll be adding 7a and 8 series as part of the update

then after I shift all current 20 devices to 21 I'll have to make a plan to start going through this backlog of requests

**Comment by yrliet** [comment]
Hi!
CalyxOS has been released for the Fairphone 5, but I still prefer DOS by far.
Whenever (if) you get the opportunity to work on it, I can try builds and provide logs (if that helps of course).
I can also try locking the bootloader "for science" once a new security patch is available :eyes: :

**Comment by cryptoo01** [comment]
Hello dear, SkewedZeppelin!
I have previously asked you to add Redmi Note 9 Pro (joyeuse) support
https://github.com/Divested-Mobile/DivestOS-Build/issues/68#issuecomment-1330751564
https://github.com/Divested-Mobile/DivestOS-Build/issues/68#issuecomment-1336105473
The device is currently supported by lineageos_21
https://wiki.lineageos.org/devices/miatoll/variant3/
With deep respect for your work and understanding of workload, please pay attention to this device, add its support to DivestOS in the near future.
Thank you very much!

**Comment by VasilKotsov** [comment]
Can please add support for Motorola Edge 40 Pro RTWO its supported by LineageOS.
Thank you.

**Comment by d4f5409d** [comment]
> Hi!
> CalyxOS has been released for the Fairphone 5, but I still prefer DOS by far.
> Whenever (if) you get the opportunity to work on it, I can try builds and provide logs (if that helps of course).
> I can also try locking the bootloader "for science" once a new security patch is available :eyes: : 
> 

I have a sinister feeling we won't get DOS for the Fairphone 5. I'll have to go for the Nothing Phone 2.

**Comment by SkewedZeppelin** [comment]
@d4f5409d 
work on dos21 already started, but there is a lot of churn from qpr 2 and 3 causing breakage upstream lineage that they're still working through.

**Comment by d4f5409d** [comment]
> @d4f5409d 
> work on dos21 already started, but there is a lot of churn from qpr 2 and 3 causing breakage upstream lineage that they're still working through.
> FP5 will be added once dos21 stabilizes.

Thank you for your answer, sir.

**Comment by CueHD** [comment]
Two addition requests:

LG V30
model joan
Latest LineageOS: [lineage-21.0-20240616](https://download.lineageos.org/devices/joan/builds)

LG G8
model alphaplus
Current LineageOS: 21 https://download.lineageos.org/devices/alphaplus/builds

**Comment by ethical-haquer** [comment]
Samsung Galaxy A52 4G
Codename: a52q
[Has LineageOS 21 support](https://wiki.lineageos.org/devices/a52q/).

**Comment by orkutmuratyilmaz** [comment]
hello,

is there a URL for vayu 20 test release?

best,
Orkut

**Comment by 4mede** [comment]
Moto g power 2021 (borneo). Recently got official Lineage.

**Comment by lepras** [comment]
> Also this is not a phone but could you add a SM-T580 (gtaxlwifi). [Not officially supported ](https://forum.xda-developers.com/t/lineageos-18-1-for-sm-t580-gtaxlwifi-and-sm-p580-gtanotexlwifi.4274293/) (lin18)

Any hope on this one?

**Comment by IbrahimAkhii** [comment]
I would like to see a Build for the Xiaomi POCO X3 NFC since it is based on Android 14 and supported by LineageOS and e/OS. And also iódeOS in 2023.

Here is the Device: https://wiki.lineageos.org/devices/surya/

I hope it will be possible! Thanks!

**Comment by jordan03819** [comment]
pdx203, pdx206, pdx214, pdx215, pdx234, all have official lineage 21 support

**Comment by Jeffrey2307** [comment]
XIAOMI POCO X3 NFC (Has LineageOS 21 & Android 14 Official Support)

I saw a few people asking for this Device too. Will there be a Build?

**Comment by Rubiginosa** [comment]
> pdx203, pdx206, pdx214, pdx215, pdx234, all have official lineage 21 support

These devices are also no longer receiving security updates from Sony, so a secure OS option would be great.

**Comment by KlausHans** [comment]
> Any plans on supporting the Pixel 8 (Pro) with the upcoming 21.0 builds?

Yes, please!

**Comment by foxierra** [comment]
> @d4f5409d work on dos21 already started, but there is a lot of churn from qpr 2 and 3 causing breakage upstream lineage that they're still working through. FP5 will be added once dos21 stabilizes.

Is there any update about it yet ? Thanks in advance !
